homeassistant/components/august/camera.py

```diff
@@ -22,7 +22,10 @@ async def async_setup_entry(
 ) -> None:
     """Set up August cameras."""
     data: AugustData = hass.data[DOMAIN][config_entry.entry_id]
-    session = aiohttp_client.async_get_clientsession(hass)
+    # Create an aiohttp session instead of using the default one since the
+    # default one is likely to trigger august's WAF if another integration
```

Comment:
Why is the default triggering WAF?

Reply:
I don't know what August has setup in their rule-set (and I don't expect them to tell us).   We had a similar problem with the original tesla integration a while back and we never were able to figure out exactly which cookie or behavior in the session triggered Telsa's WAF but a separate session solved it.

Reply:
Interesting 🤔

Reply:
As discussed on Discord, this is interesting to research at some point. For now, going o merge it in to fix the issue at hand.